core-bundle/src/Resources/contao/library/Contao/Controller.php

```diff
@@ -211,7 +211,7 @@ public static function getTemplateGroup($strPrefix, array $arrAdditionalMapper=a
 					}
 				}
 
-				$arrTemplates[$strTemplate][] = $GLOBALS['TL_LANG']['MSC']['global'];
+				$arrTemplates[$strTemplate][] = $GLOBALS['TL_LANG']['MSC']['global'] ?? 'Global';
```

Comment:
```suggestion
				$arrTemplates[$strTemplate][] = $GLOBALS['TL_LANG']['MSC']['global'] ?? 'global';
```

Reply:
why? It's the title of the global template `optgroup`?

Reply:
The default string for `$GLOBALS['TL_LANG']['MSC']['global']` is also `global`, therefore I figured it should be consistent. What do you mean by _global template optgroup_?